### PR TITLE
Customizable wpt options and profile in workflows

### PR DESCRIPTION
**Pull request by sagudev**
Split off #30694 

This PR is reviewable per commit.

also includes a fix for try labels that was caused by https://github.com/servo/servo/pull/30903

---

- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors
- [x] These changes do not require tests because it's CI

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -85,7 +98,7 @@ jobs:
     if: ${{ contains(fromJson(needs.decision.outputs.configuration).platforms, 'windows') }}
     uses: ./.github/workflows/windows.yml
     with:
-      profile: "release"
+      profile: ${{ fromJson(needs.decision.outputs.configuration).profile }}${{ fromJson(needs.decision.outputs.configuration).profile }}
```
Comment (mrobinson) [inline .github/workflows/main.yml:101]: Looks like this `${{ fromJson(needs.decision.outputs.configuration).profile }}` was inadvertently doubled here?

File: .github/workflows/mac.yml [hunk 2]
```diff
@@ -7,6 +7,10 @@ on:
         required: false
         default: "release"
         type: string
+      wpt:
+        default: ""
+        required: false
+        type: string
       wpt-layout:
         required: false
         type: string
```
Comment (mrobinson) [inline .github/workflows/mac.yml:16]: This becomes a bit confusing with `wpt`, `wpt-layout`, and `wpt-sync` as it isn't clear what each of these does in an obvious way. Suggestion:

`wpt`: `wpt-tests-to-run`
`wpt-layout`: `wpt-layout-system`
`wpt-sync`: `wpt-sync-tests-from-upstream`

**Review by mrobinson — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (sagudev) [reply .github/workflows/main.yml:101]: This is my touchpad, will fix it.

**Review by mrobinson — APPROVED** [review]

File: .github/workflows/scheduled-wpt-import.yml [hunk 3]
```diff
@@ -67,7 +67,7 @@ jobs:
         run: |
           BODY=$(cat <<EOF
           Automated downstream sync of changes from upstream as of ${{ env.CURRENT_DATE }}
-          [no-wpt-sync]
+          [no-wpt-sync-from-upstream]
```
Comment (mrobinson) [inline .github/workflows/scheduled-wpt-import.yml:70]: This one shouldn't be changed, I think.

**Review by mrobinson — CHANGES_REQUESTED** [review]

**Review by mrobinson — APPROVED** [review]

**Pull request merged**